variables.tf

```
@@ -0,0 +1,319 @@
+}
+
+variable "policy_arn" {
+  type        = "string"
+  default     = "arn:aws:iam::aws:policy/AdministratorAccess"
+  description = "Permission to grant to atlantis server"
+}
+
+variable "kms_key_id" {
+  type        = "string"
+  default     = ""
+  description = "KMS key alias used to encrypt SSM SecureString parameters"
+}
+
+variable "webhook_secret_length" {
+  default     = 32
+  description = "GitHub webhook secret length"
+}
+
+variable "webhook_events" {
+  type        = "list"
+  description = "A list of events which should trigger the webhook."
+
+  default = [
+    "issue_comment",
+    "pull_request",
+    "pull_request_review",
+    "pull_request_review_comment",
+    "push",
+  ]
+}
+
+variable "ssh_private_key_name" {
+  type        = "string"
+  default     = "atlantis_ssh_private_key"
+  description = "Atlantis SSH private key name - used for namespacing in SSM"
+}
+
+variable "ssh_public_key_name" {
+  type        = "string"
+  default     = "atlantis_ssh_public_key"
+  description = "Atlantis SSH public key name - used for namespacing in SSM"
+}
+
+variable "vpc_id" {
+  type        = "string"
+  description = "VPC ID for the ECS Cluster"
+}
+
+variable "alb_listener_arns" {
+  type        = "list"
+  default     = []
+  description = "A list of all the ALB listener ARNs"
+}
+
+variable "alb_ingress_paths" {
+  type        = "list"
+  default     = ["/*"]
+  description = "Path pattern to match (a maximum of 1 can be defined), at least one of hosts or paths must be set"
+}
+
+variable "alb_name" {
+  type        = "string"
+  description = "The Name suffix of the ALB"
+}
+
+variable "alb_arn_suffix" {
+  type        = "string"
+  description = "The ARN suffix of the ALB"
+}
+
+variable "alb_target_group_alarms_alarm_actions" {
+  type        = "list"
+  description = "A list of ARNs (i.e. SNS Topic ARN) to execute when ALB Target Group alarms transition into an ALARM state from any other state."
+  default     = []
+}
+
+variable "alb_target_group_alarms_ok_actions" {
+  type        = "list"
+  description = "A list of ARNs (i.e. SNS Topic ARN) to execute when ALB Target Group alarms transition into an OK state from any other state."
+  default     = []
+}
+
+variable "alb_target_group_alarms_insufficient_data_actions" {
+  type        = "list"
+  description = "A list of ARNs (i.e. SNS Topic ARN) to execute when ALB Target Group alarms transition into an INSUFFICIENT_DATA state from any other state."
+  default     = []
+}
+
+variable "alb_dns_name" {
+  type        = "string"
+  description = "DNS name of ALB"
+}
+
+variable "alb_zone_id" {
+  type        = "string"
+  description = "The ID of the zone which ALB is provisioned"
+}
+
+variable "ecs_cluster_name" {
+  type        = "string"
+  description = "Name of the ECS cluster to deploy Atlantis"
+}
+
+variable "ecs_cluster_arn" {
+  type        = "string"
+  description = "ARN of the ECS cluster to deploy Atlantis"
+}
+
+variable "vpc_default_security_group_id" {
```

```suggestion
variable "vpc_security_group_id" {
```

Why it should be the default SG and not any SG?